lms/envs/aws.py

```diff
@@ -89,6 +89,7 @@
 DEFAULT_PRIORITY_QUEUE = 'edx.{0}core.default'.format(QUEUE_VARIANT)
 LOW_PRIORITY_QUEUE = 'edx.{0}core.low'.format(QUEUE_VARIANT)
 HIGH_MEM_QUEUE = 'edx.{0}core.high_mem'.format(QUEUE_VARIANT)
+GRADES_BACKFILL_QUEUE = 'edx.{0}core.grades_backfill'.format(QUEUE_VARIANT)
```

Comment:
Unless you're cherry-picking something from upstream, Stanford-specific settings _always_ go in `openedx/stanford/lms/envs/*.py`